metals/src/main/scala/scala/meta/internal/metals/CodeLensProvider.scala

```diff
@@ -42,12 +58,11 @@ final class DebugCodeLensProvider(
       classes: BuildTargetClasses.Classes
   ): Seq[l.CodeLens] = {
     semanticdbs.textDocument(path).documentIncludingStale match {
-      case _ if classes.isEmpty =>
+      case None =>
```

Comment:
need to revert that

Reply:
actually this is correct, even if classes.isEmpty we might still need generate lenses for methods